templates/default/sql/dml/2.1.0.sql.erb

```diff
@@ -1 +1,2 @@
-REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("fs_job_util", "hdfs:///user/<%= node['hadoop_spark']['user'] %>/hsfs_utils-<%= node['hadoop_spark']['hsfs']['utils']['version'] %>.py");
+REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("fs_py_job_util", "hdfs:///user/<%= node['hadoop_spark']['user'] %>/hsfs_utils-<%= node['hadoop_spark']['hsfs']['utils']['version'] %>.py");
```

Comment:
This is not how these files work, you need to put your changes in the 3.0.0 files.